### PR TITLE
Changing max_daily_rate based on Exchange

### PR DESCRIPTION
**Pull request by chriscrutt**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

Here is a pull request to solve the problem of `max_daily_rate`. As you know, Poloniex has a max rate of 5%. Bitfinex however... does not. The bot checks to see if they're using Poloniex or Bitfinex; if it's using Bitfinex it sets the max and min daily rate range from 0.003% to infinity. It also changes the default value of `max_daily_rate` to either 5% or infinity depending on exchange.

Here is my code and how I fixed it. It seems to be running smoothly so far, I'm waiting for my current period to be up and then see if the bot glitches or not.

### CODE REVIEW

File: modules/Lending.py [hunk 1]
```diff
@@ -29,7 +33,6 @@
 coin_cfg = {}
 dry_run = 0
 transferable_currencies = []
-currencies_to_analyse = []
```
Comment (rnevet) [inline modules/Lending.py]: Is this related to the change?

Comment (chriscrutt) [reply modules/Lending.py]: I dont know exactly why my file had `exchange, analysis_method, currencies_to_analyse` taken out and replaced with `exchange, analysis_method`, maybe I had a slightly older file? Same with `if Analysis and cur in currencies_to_analyse:` and `if Analysis:` 

Sorry. Also I just realized that in my default.cfg I had `max_daily_rate` uncommented... and when I commented it to set it to the infinite default value the bot ceased to work.

**Comment by chriscrutt** [comment]
I screwed some stuff up I was not aware of and the bot didn't show.
